backend/app/views/questions/show.slim

```diff
@@ -0,0 +1,32 @@
+h1
+  = @question.name
+h5
+  | This is a 
+  = @question.question_type.name.downcase
+  |  question.
+ul
+  - @question.options.each do |option|
+    - if @question.single_choice?
+      |  <input type='radio' id="
```

Comment:
ops, looks like there are a few thing incorrect in this syntax, please doublecheck this file

Reply:
Hmm 🤔